encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -64,7 +64,7 @@ package <$pkgname>
 // Interface is the server-side interface for the <.Service.Name> service.
 type Interface interface {
 	<if .Parent>
-		<$parentPath := printf "%s/yarpc/%sserver" .ParentModule.ImportPath (lower .Parent.Name)>
+		<$parentPath := printf "%s/%sserver" .ParentModule.ImportPath (lower .Parent.Name)>
```

Comment:
File name or package name stutter doesn't affect users directly much. We named
the packages ` ${service}server` and `${service}client` so that users didn't
have to do named imports all the time. Calling them just `server` and `client`
means you *have* to do a named import if you are referencing more than one
service from the same file.

    import kvclient ".../keyvalue/client"
    import healthclient ".../health/client"
    
The other option we considered was naming the YARPC package just `$service`.
We chose not to do this for the same reason: avoiding named imports. The
service name is often the same as the Thrift file name, so with `foo/foo`
providing YARPC types and `foo/` providing types defined in the Thrift file,
you would have to use named imports again.